pkg/util/cryptomaterial/certpaths.go

```diff
@@ -11,8 +13,9 @@ const (
 	ClientCertFileName = "client.crt"
 	ClientKeyFileName  = "client.key"
 
-	ClientCAValidityDays   = 60
-	ClientCertValidityDays = 30
+	ClientCAValidityDays                  = 60
+	ClientCertValidityDays                = 30
+	AdminKubeconfigClientCertValidityDays = 365 * 10
```

Comment:
Is this documented/required anywhere?

Reply:
the documentation is to be found here: https://github.com/openshift/api/blob/master/tls/docs/kube-apiserver%20Client%20Certificates/README.md#admin-kubeconfig-signer

I noticed that the client CAs are actually a bit more diverse in their validity configurations, I will be fixing that up in a follow up along with a refactoring of how this works.